packages/vector_graphics_compiler/test/parser_test.dart

```diff
@@ -22,38 +22,50 @@ void main() {
     expect(instructions.paints, const <Paint>[
       Paint(
           blendMode: BlendMode.srcOver,
-          stroke: Stroke(color: Color(0xffffee44), width: 3.0)),
+          stroke: Stroke(
+              color: Color(0xffffee44), join: StrokeJoin.round, width: 3.0)),
```

Comment:
Here and below: 

This is happening because inheritence is now working from the `<svg>` root element, where before it didn't. This is covered in a different test that specifically targets that as well.